getting-started/QUICKSTART.md

```diff
@@ -6,7 +6,7 @@ _**Navigation**_
 - [02.SAMPLES](./SAMPLES.md)
 ___
 
-In this quickstart we will show you how to get the Echo bot up and running.
+In this quickstart we will show you how to get the Echo bot up and running. The Echo bot echoes back messages sent to it while keeping track of the number of messages sent by the user.
```

Comment:
```suggestion
In this quickstart we will show you how to get the Echo bot up and running. The EchoBot echoes back messages sent to it while keeping track of the number of messages sent by the user.
```

Reply:
The name "Echo bot" has been used instead of "EchoBot" in rest of the getting started docs, so we can just keep it as it is.

Reply:
After discussion with Corina, I just we should update it to "The Echo Bot".

Reply:
I updated the naming to "The Echo Bot" in these getting started readme docs.